R/FilterStateDatettime.R

```diff
@@ -289,6 +292,14 @@ DatetimeFilterState <- R6::R6Class( # nolint
       private$is_choice_limited <- (any(xl < choices[1L], na.rm = TRUE) | any(xl > choices[2L], na.rm = TRUE))
       invisible(NULL)
     },
+
+    get_metadata_expression = function(dataname) {
+      if (missing(dataname)) dataname <- private$dataname
+      varname <- private$get_varname_prefixed(dataname)
+      selected <- private$get_selected()
+      sprintf("%1$s => %2$s & %1$s <= %3$s", deparse1(varname), selected[1], selected[2])
+    },
```

Comment:
what is the difference between this and `get_call`?

Reply:
no Inf and NA information